projects/step-core/src/lib/modules/basics/components/time-input/operation-per-time-input.component.ts

```diff
@@ -0,0 +1,34 @@
+import { TimeUnit } from '../../shared/time-unit.enum';
+import { BaseTimeConverterComponent } from './base-time-converter.component';
+import { Component, ViewEncapsulation } from '@angular/core';
+import { NgControl } from '@angular/forms';
+
+@Component({
+  selector: 'step-operation-per-time-input',
+  templateUrl: './base-time-converter.component.html',
+  styleUrls: ['./base-time-converter.component.scss'],
+  encapsulation: ViewEncapsulation.None,
+})
+export class OperationPerTimeInputComponent extends BaseTimeConverterComponent {
```

Comment:
For me the naming doesn't make sense. I would name it "PerTimeUnitInputComponent" (etc.)

Reply:
Done